### ISSUE TITLE
How do you import midi files with musconv?

### ISSUE THREAD

**Issue by JeoLeaf** [captainsouthbird/SMB3Utilties#2]
I got Nodice working but I can not figure out how musconv works. I run it in the terminal with "musconv import midi.mid asm.asm" But I get an IndexOffs error.

**Comment by VallenateroBrony2017** [comment]
Creo que es imposible para mí importar estos archivos. ¿Cómo puedo importar estos midis a Super Mario Bros. 3?

**Comment by VallenateroBrony2017** [comment]
Yo todavía no sé como introducir estos midis al Super Mario Bros. 3

**Comment by captainsouthbird** [comment]
MusConv is pretty esoteric. Honestly it'd probably require its own tutorial video and even then is pretty clumsy and fragile. I do have documentation such as it is that I wrote here: http://sonicepoch.com/sm3mix/disassembly.html#MUSCONV

I'm not personally planning on developing the tool any further... it did its job for as much as I cared to use it. I did add MIDI files I used in 3Mix to its repo just for the sake of example. Other than that, you're on your own... maybe someone out there will feel compelled to make a better tool, but I'd rather work on newer projects.
